### PR TITLE
add date range slider to data catalog

### PR DESCRIPTION
**Pull request by bnfish**
filtering by date range not functional yet; start and end date parameters will need to be added to the streams service as discussed with @FBRTMaka

### CODE REVIEW

**Comment by FBRTMaka** [comment]
@bnfish nice Ben!! Thanks!!

File: ooiui/templates/science/streams.html [hunk 1]
```diff
@@ -21,6 +22,14 @@
...
 
   <script src="/js/compiled/svgplot.js" type="text/javascript"></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderMouseTouch.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderDraggable.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderBar.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderHandle.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSliderLabel.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQRangeSlider.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQDateRangeSliderHandle.min.js'></script>
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jQRangeSlider/5.7.1/jQDateRangeSlider.min.js'></script>
```
Comment (birdage) [inline ooiui/templates/science/streams.html]: we should use local refs not CDN links

Comment (FBRTMaka) [inline ooiui/templates/science/streams.html]: @birdage Yup! iterative steps will be made to get this to be locally hosted.  Function before form at the moment.

Comment (birdage) [inline ooiui/templates/science/streams.html]: we cant let it be forgotten, as we saw recently if the CDN or the connection to the CDN goes down, the UI will die. Very simple to add to bower and get it working not on a CDN, suggest we look at this next week.

Comment (FBRTMaka) [inline ooiui/templates/science/streams.html]: Absolutely right, it's not for next week (this week) as most people are out.  But it will be done!

**Comment by DanielJMaher** [comment]
@maka-io OBE by your code

**Pull request closed without merging**